src/modules/claim/claim.dto.ts

```diff
@@ -157,6 +158,10 @@ export class ClaimRejectionDTO implements IClaimRejection {
   @IsBoolean()
   @ApiProperty()
   isRejected: boolean;
+
+  @IsEnum(RejectionReason)
```

Comment:
Let's keep it simple. Maybe a max-length though would be good though. Maybe 200 char or something?
```suggestion
  @IsString()
```